study.md

```diff
@@ -46,14 +46,24 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+I don't think it's anything special... just adds an exclamation mark after the string.
```

Comment:
When you call a method with the `!` after that means it mutates the original thing it operates on.